students/kyle_odonnell/Lesson04/mailroom.py

```diff
@@ -0,0 +1,258 @@
+# ------------------------------------------------------------------------ #
+# Title: mailroom.py
+# Description: Assignment for Lesson04
+# KODonnell,10.25.2020,created script
+# KODonnell,11.05.2020 added switch dict menu
+# KODonnell,11.07.2020 added function to generate letters
+# KODonnell,11.10.2020 split file writing functions into multiple funcs
+# ------------------------------------------------------------------------- #
+
+
+# Data -------------------------------------------------------------------- #
+
+# Database for donor information
+donor_list = [
+    {"Donor Name": "William Gates, III", "Total Given": 653784.49, "Num Gifts": 2, "Average Gift": 326892.24},
+    {"Donor Name": "Mark Zuckerberg", "Total Given": 16396.10, "Num Gifts": 3, "Average Gift": 5465.37},
+    {"Donor Name": "Jeff Bezos", "Total Given": 877.33, "Num Gifts": 1, "Average Gift": 877.33},
+    {"Donor Name": "Paul Allen", "Total Given": 708.42, "Num Gifts": 3, "Average Gift": 236.14},
+    {"Donor Name": "Elon Musk", "Total Given": 50.00, "Num Gifts": 2, "Average Gift": 25.00}
+]
```

Comment:
in your this lesson you are expected to convert main data structure to be a dictionary - you are currently still using a list.